src/presence_analyzer/static/mean_time_weekday.html

```diff
@@ -96,28 +29,26 @@
                     dropdown.show();
                     loading.hide();
                 });
-                $('#user_id').change(function(){
-                    var selected_user = $("#user_id").val();
-                    var chart_div = $('#chart_div');
+                $('#user_id').change(function() {
+                    var selected_user = $("#user_id").val(),
+                        chart_div = $('#chart_div');
```

Comment:
camelCase in js, jquery selectors var shoud lbe proceed by $ to indicate it's a jquery selector, so `$chartDiv`